easybuild/framework/easyconfig/types.py

```diff
@@ -116,3 +110,34 @@ def convert_value_type(val, typ):
         raise EasyBuildError("No conversion function available (yet) for target type %s", typ)
 
     return res
+
+def to_dict(tc):
+    """
+    Convert a toolchain string "foo, v0" to a dictionary {'name':'foo', 'version':'v0'}
+
+    @param toolchain_str: a toolchain string
```

Comment:
or list, and there's no such thing as `toolchain_str`

according to PEP8, two-letter variables are not a good idea, so maybe use `tcspec`?